Modules/Loadable/Markups/MRML/vtkMRMLMarkupsAngleNode.cxx

```diff
@@ -45,18 +47,179 @@ vtkMRMLMarkupsAngleNode::~vtkMRMLMarkupsAngleNode() = default;
 void vtkMRMLMarkupsAngleNode::WriteXML(ostream& of, int nIndent)
 {
   Superclass::WriteXML(of,nIndent);
+
+  vtkMRMLWriteXMLBeginMacro(of);
+  vtkMRMLWriteXMLEnumMacro(angleMode, AngleMode);
+  vtkMRMLWriteXMLVectorMacro(surfaceDistanceWeightingFunction, OrientationRotationAxis, double, 3);
+  vtkMRMLWriteXMLEndMacro();
 }
 
 //----------------------------------------------------------------------------
 void vtkMRMLMarkupsAngleNode::ReadXMLAttributes(const char** atts)
 {
   Superclass::ReadXMLAttributes(atts);
+
+  vtkMRMLReadXMLBeginMacro(atts);
+  vtkMRMLReadXMLEnumMacro(angleMode, AngleMode);
+  vtkMRMLReadXMLVectorMacro(surfaceDistanceWeightingFunction, OrientationRotationAxis, double, 3);
+  vtkMRMLReadXMLEndMacro();
+}
+
+//----------------------------------------------------------------------------
+void vtkMRMLMarkupsAngleNode::CopyContent(vtkMRMLNode* anode, bool deepCopy/*=true*/)
+{
+  MRMLNodeModifyBlocker blocker(this);
+  Superclass::CopyContent(anode, deepCopy);
+
+  vtkMRMLCopyBeginMacro(anode);
+  vtkMRMLCopyEnumMacro(AngleMode);
+  vtkMRMLCopyVectorMacro(OrientationRotationAxis, double, 3);
+  vtkMRMLCopyEndMacro();
 }
 
 //----------------------------------------------------------------------------
 void vtkMRMLMarkupsAngleNode::PrintSelf(ostream& os, vtkIndent indent)
 {
   Superclass::PrintSelf(os,indent);
+
+  vtkMRMLPrintBeginMacro(os, indent);
+  vtkMRMLPrintEnumMacro(AngleMode);
+  vtkMRMLPrintVectorMacro(OrientationRotationAxis, double, 3);
+  vtkMRMLPrintEndMacro();
+}
+
+//----------------------------------------------------------------------------
+int vtkMRMLMarkupsAngleNode::GetAngleMode()
+{
+  return this->AngleMode;
```

Comment:
Modified() call is missing. Also, if you just need to get/set value then use the standard get/set macro instead.